src/components/pages/species/list/filter/shared/hsv-color-picker/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { AccordionButton, AccordionIcon, AccordionItem, AccordionPanel, Box } from '@chakra-ui/react';
+import debounce from 'debounce-promise';
+import useTranslation from 'next-translate/useTranslation';
+import React from 'react';
+import { HsvColorPicker } from 'react-colorful';
+
+import useSpeciesList from '../../../use-species-list';
+
+export interface FilterHsvProps {
+    filterKey;
+    label?;
+    translateKey?;
+}
+
+export default function HsvColorFilter(props: FilterHsvProps) {
+    
+    const { addFilter, removeFilter } = useSpeciesList();
+    const { t } = useTranslation();
+    const onQuery = debounce((val) => handleOnChange(val), 200);
+
+    const handleOnChange = (val) => {
+        if (val) {
+            const { h, s, v } = val;
+            addFilter(props.filterKey, `${h},${s},${v}`);
+        } else {
+            removeFilter(props.filterKey);
+        }
+    };
+
+    return (
+        <AccordionItem>
```

Comment:
@teamdraftbox is there some change in formatting? since this doesn't look like two spaces 😅